code/modules/cm_aliens/structures/special/pylon_core.dm

```diff
@@ -151,9 +151,14 @@
 		if(linked_hive.hijack_burrowed_surge && (last_surge_time + surge_cooldown) < world.time)
 			last_surge_time = world.time
 			linked_hive.stored_larva++
+			linked_hive.hijack_burrowed_left--
 			announce_dchat("The hive has gained another burrowed larva! Use the Join As Xeno verb to take it.", src)
 			if(surge_cooldown > 30 SECONDS) //mostly for sanity purposes
 				surge_cooldown = surge_cooldown - surge_incremental_reduction //ramps up over time
+		if(linked_hive.hijack_burrowed_surge && linked_hive.hijack_burrowed_left < 1)
```

Comment:
There isn't much reason why this can't just be done underneath the if block on line 151 `if(linked_hive.hijack_burrowed_surge && (last_surge_time + surge_cooldown) < world.time)` as just `if(linked_hive.hijack_burrowed_left < 1)` tabbed over one to this right where it is now.

Reply:
Done